jina/peapods/remote.py

```diff
@@ -85,13 +95,7 @@ def __init__(self, args: Union['Namespace', Dict]):
                                                                          first_pea_args.ctrl_with_ipc)
 
     def spawn_remote(self, host: str, port: int, pod_type: str = 'cli', **kwargs) -> Optional[str]:
-        return super().spawn_remote(host, port, pod_type=pod_type)
-
-    def send_terminate_signal(self) -> None:
-        """Gracefully close this pea and release all resources """
-        if self.is_ready_event.is_set() and hasattr(self, 'ctrl_addr'):
-            send_ctrl_message(self.ctrl_addr, jina_pb2.RequestProto.ControlRequestProto.TERMINATE,
-                              timeout=self.args['peas'][0].timeout_ctrl)
+        return super().spawn_remote(host=host, port=port, pod_type=pod_type)
```

Comment:
why do we need this, just to override the default `pod_type`?

Reply:
for now, yes. we've separated into 2 classes, as `PeaAPI` and `PodAPI` might have different endpoints.